extras/jenkins/README.md

```diff
@@ -0,0 +1,23 @@
+## Directory
+
+`<repository-root>/extras/jenkins`
+
+## Purpose
+
+This directory contains a set of Jenkinsfiles that can be used with MARA. These should be considered to be in a "draft"
```

Comment:
Linking out to something that explains what a `Jenkinsfile` does would be useful. Also, you may want to rephrase this so you do not pluralize `Jenkinsfile` because that's confusing due to it being a filename convention.

Reply:
done.